src/components/ManageTeamHealth.vue

```diff
@@ -42,35 +61,81 @@ export default {
     },
 
     reset() {
-      this.$emit('reset');
+      // TODO: Reset i
       this.hasChanged = false;
     },
 
     changeStatus(team, key) {
       this.hasChanged = false;
-      if (team.criteria_values[key] === 3) {
-          team.criteria_values[key] = 1;
+      if (team.criteria_values[key] === STATUS.GREEN) {
+          team.criteria_values[key] = STATUS.RED;
       }
       else {
           team.criteria_values[key]++;
       }
       this.hasChanged = true;
     },
+
+    /**
+     * Compares a team's criteria value with the previous sprint value, if it exists, 
+     * to determine whether there was an increase or decrease. 
+     * @returns -1 for decrease, 0 for no change, 1 for increase
+     */
+    getStatusChange(team, key) {
+      // Return 0 if this is the first sprint
+      if (this.currentSprint.sprintNumber === 0) {
+        return 0;
+      }
+
+      // Get the previous sprint
+      const previousSprintNo = parseInt(this.currentSprint.sprintNumber - 1).toString() 
+      const previousSprint = this.getSprintByNumber(previousSprintNo);
+
+      if (!_.isUndefined(previousSprint)) {
+        // Get the two sprint team values to compare
+        const currentVal = team.criteria_values[key];
+        const previousVal = previousSprint.teams[team.id].criteria_values[key];
+
+        if (currentVal === previousVal) {
+          return 0; // No change
+        }
+        else if (currentVal > previousVal) {
+          return 1; // Value increased
+        }
+        else {
+          return -1; // Value decreased
+        }
+      }
+      return 0;
+    },
+
+    getSprintByNumber(sprintNo) {
```

Comment:
why don't you change this to `getSprintByNumber(sprintNumber)` then you can do `_.find(this.project.sprints, { sprintNumber })`? Literally does not matter at all, just came to mind haha.

Reply:
That's nicer, I'll do that 👍